index.html

```diff
@@ -5,38 +5,183 @@
+          <a href="#" class="bi bi-twitter"></a>
+        </span>
+        <span class="upper-menu__links">
+          <a href="#">English</a>
+          <a href="#">My Page</a>
+          <a href="#">Logout</a>
+        </span>
+      </span>
+      <div class="links">
+        <a class="logo" href="#"><img src="./images/logo-rock2.jpg" alt="header icon"></a>
+        <nav>
+          <ul class="menu__list">
+            <li class="menu__item menu__about"><a href="#">About</a></li>
+            <li class="menu__item"><a href="#">Program</a></li>
+            <li class="menu__item"><a href="#">Join</a></li>
+            <li class="menu__item"><a href="#">Sponsor</a></li>
+            <li class="menu__item"><a href="#">News</a></li>
+            <li class="menu__item mark"><a href="#">Century Concert</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+      <section class="cover" id="cover">
+        <h3>"Welcome music lovers"</h3>
+        <h1>THE CONCERT OF THE CENTURY <br>THE MOST SPECTACULAR MEETING OF POP AND ROCK IN 2022</h1>
+        <p>A happy meeting with your favorite bands that will leave an unforgettable memory with a perfect sound that will reverberate throughout Colombia, only at the Arena Plaza in Bogota in October.</p>
+        <h2>2022.10.21(FRI) - 22(SAT)</h2>
+        <h4>Bogotá, Cota, Cundinamarca</h4>
+      </section>
+      <section class="program">
+        <h3>Main Program</h3>
+        <ul class="event-fase">
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <span>Opening</span>
+            <p>Inauguration of the event with the general director of the festival.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <span>Musical History</span>
+            <p>Here you will find the main milestones in the musical journey of our guest bands.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <span>Store</span>
+            <p>Only on this site can you buy souvenirs and musical products from your favorite band.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <span>Main Event</span>
+            <p>You can listen to all the bands Friday and Saturday from 4 pm and we will finish at 11 pm.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <span>Closure</span>
+            <p>Each of the bands in a special place will be autographing your favorite memories.</p>
+          </li>
+        </ul>
+        <a class="btn" href="#">Join to Concert of the Century</a>
+        <a class="btn-program" href="#main-page">SEE THE WHOLE PROGRAM</a>
+      </section>
+      <div id="featured">Featured Bands</div>
+      <section class="bands" id="bands" >
+        
+      </section>
```

Comment:
### Missing Button
![missing-extra-button](https://user-images.githubusercontent.com/93039359/181714833-13e95cd5-ef6b-4dac-a0a7-29b671924365.png)

- [ ] You are missing this See more button in your design. Please add it :+1: You don't need to add the functionality of this button because it is an optional requirement :relaxed::